products/kuma.md

```diff
@@ -0,0 +1,51 @@
+---
+title: kuma
+category: app
+tags: cncf configuration-management
```

Comment:
```suggestion
tags: cncf
```

The set of allowed tag is restricted, see https://endoflife.date/contribute. And we don't have a `configuration-management` tag for now.